packages/wallet/frontend/src/components/dialogs/CreateWalletAddressDialog.tsx

```diff
@@ -67,13 +63,12 @@ export const CreateWalletAddressDialog = ({
                   as="h3"
                   className="text-center text-2xl font-bold"
                 >
-                  Create {titleExtra} Payment Pointer
+                  Create Payment Pointer
```

Comment:
The `TemporaryWMNotice` component can be removed as well:
https://github.com/interledger/testnet/blob/1aaf8ea3092de5dc1e3fc913a8e34ecf75968a56/packages/wallet/frontend/src/components/dialogs/CreateWalletAddressDialog.tsx#L65-L68